.github/actions/setup-solc-select/action.yaml

```diff
@@ -0,0 +1,32 @@
+name: Setup Solc Select
+description: Installs Solc Select, required versions and selects the version to use. Requires Python 3.6 or higher.
+inputs:
+  to_install:
+    description: Comma-separated list of solc versions to install
+    required: true
+  to_use:
+    description: Solc version to use
+    required: true
+
+runs:
+  using: composite
+  steps:
+    - name: Install solc-select and solc
+      shell: bash
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y python3-pip
```

Comment:
```suggestion
```

pip3 should already be installed, see the list of installed packages on the [ubuntu-22.04 runner image](https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2204-Readme.md).